package.json

```diff
@@ -14,6 +14,7 @@
     "@fullcalendar/daygrid": "^6.1.10",
     "@fullcalendar/interaction": "^6.1.10",
     "@fullcalendar/react": "^6.1.10",
+    "@tldraw/tldraw": "^2.0.0-beta.3",
```

Comment:
We should watch this for when it's out of beta. Probably works fine now but beta stuff is ehhhh, we can leave it